pkg/catalog/manager/sync.go

```diff
@@ -45,31 +123,5 @@ func (m *Manager) Sync(key string, obj *v3.Catalog) error {
 		}()
 		return nil
 	}
-
-	// always get a refresh catalog from etcd
-	catalog, err := m.catalogClient.Get(key, metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-
-	repoPath, commit, err := m.prepareRepoPath(*catalog)
-	if err != nil {
-		v3.CatalogConditionRefreshed.False(catalog)
-		v3.CatalogConditionRefreshed.ReasonAndMessageFromError(catalog, err)
-		m.catalogClient.Update(catalog)
-		return err
-	}
-
-	if commit == catalog.Status.Commit {
-		logrus.Debugf("Catalog %s is already up to date", catalog.Name)
-		if v3.CatalogConditionRefreshed.IsUnknown(catalog) {
-			v3.CatalogConditionRefreshed.True(catalog)
-			v3.CatalogConditionRefreshed.Reason(catalog, "")
-			m.catalogClient.Update(catalog)
-		}
-		return nil
-	}
-
-	logrus.Infof("Updating catalog %s", catalog.Name)
-	return m.traverseAndUpdate(repoPath, commit, catalog)
+	return nil
```

Comment:
This common code is handling delete of the Catalog's templates and templateVersion objects. After the delete, the outer caller method 'sync' - should return nil.

Reply:
Also consider renaming the method to "handleDelete()" instead of "commonSync()"?